lib/provisioning/ProvisionDispatcher.js

```diff
@@ -76,6 +76,22 @@ class ProvisionDispatcher {
         ], cb);
     }
 
+    checkState(cb, autoTrigger) {
+        this._onLeaderChange(autoTrigger);
+        this._onOwnerChange(autoTrigger);
+        this._onMyselfChange(cb, autoTrigger);
+        this._onProvisionChange(autoTrigger);
+        return undefined;
+    }
+
+    periodicCheckState(cb) {
+        setTimeout(() => {
+            this._log.debug('periodic check state');
+            this.checkState(cb, false);
+            this.periodicCheckState(cb);
+        }, 60000);
```

Comment:
Could be better to have this constant defined at the top of the file to make it more visible (ideally we would have it an optional parameter of the class, but it may need some refactor so not pushing for it).